doc/cephfs/client-config-ref.rst

```diff
@@ -32,190 +32,195 @@ To check configured options use the `config get` command::
+``client_oc_max_objects``
 
 :Description: Set the maximum number of objects in the object cache.
 :Type: Integer
 :Default: ``1000``
 
-``client oc size``
+``client_oc_size``
 
 :Description: Set how many bytes of data will the client cache.
 :Type: Integer
 :Default: ``209715200`` (200 MB)
 
-``client oc target dirty``
+``client_oc_target_dirty``
 
 :Description: Set the target size of dirty data. We recommend to keep this number low.
 :Type: Integer
 :Default: ``8388608`` (8MB)
 
-``client permissions``
+``client_permissions``
 
 :Description: Check client permissions on all I/O operations.
 :Type: Boolean
 :Default: ``true``
 
-``client quota``
+``client_quota``
 
 :Description: Enable client quota checking if set to ``true``.
 :Type: Boolean
 :Default: ``true``
 
-``client quota df``
+``client_quota_df``
 
 :Description: Report root directory quota for the ``statfs`` operation.
 :Type: Boolean
 :Default: ``true``
 
-``client readahead max bytes``
+``client_readahead_max_bytes``
 
 :Description: Set the maximum number of bytes that the client reads ahead for future read operations. Overridden by the ``client_readahead_max_periods`` setting.
 :Type: Integer
 :Default: ``0`` (unlimited)
 
-``client readahead max periods``
+``client_readahead_max_periods``
 
 :Description: Set the number of file layout periods (object size * number of stripes) that the client reads ahead. Overrides the ``client_readahead_max_bytes`` setting.
 :Type: Integer
 :Default: ``4``
 
-``client readahead min``
+``client_readahead_min``
 
 :Description: Set the minimum number bytes that the client reads ahead.
 :Type: Integer
 :Default: ``131072`` (128KB)
 
-``client reconnect stale``
+``client_reconnect_stale``
 
 :Description: Automatically reconnect stale session.
 :Type: Boolean
 :Default: ``false``
 
-``client snapdir``
+``client_snapdir``
 
 :Description: Set the snapshot directory name.
 :Type: String
 :Default: ``".snap"``
 
-``client tick interval``
+``client_tick_interval``
 
 :Description: Set the interval in seconds between capability renewal and other upkeep.
 :Type: Float
 :Default: ``1.0`` (seconds)
 
-``client use random mds``
+``client_use_random_mds``
 
 :Description: Choose random MDS for each request.
 :Type: Boolean
 :Default: ``false``
 
-``fuse default permissions``
+``fuse_default_permissions``
 
 :Description: When set to ``false``, ``ceph-fuse`` utility checks does its own permissions checking, instead of relying on the permissions enforcement in FUSE. Set to ``false`` together with the ``client acl type=posix_acl`` option to enable POSIX ACL.
 :Type: Boolean
 :Default: ``true``
 
-``fuse max write``
+``fuse_max_write``
 
-:Description: Set the maximum number of bytes in a single write operation.  Because the FUSE default is 128kbytes, SO fuse_max_write default set to 0(The default does not take effect)
+:Description: Set the maximum number of bytes in a single write operation.
+              A value of 0 indicates no change, ie. the
```

Comment:
```suggestion
              A value of 0 indicates no change. For example, the
```

Reply:
I was compelled to clarify this.  I struggled with this phrasing for a while, to be sure I understood what was being expressed.  I don't think "For example" is the best in this case, since we're discussing only one specific value for this one setting that is notable.  I'm pushing an alternative, PTAL